deploy/docker/seqr/Dockerfile

```diff
@@ -102,7 +43,7 @@ RUN git clone -q --branch $SEQR_GIT_BRANCH $SEQR_REPO
 WORKDIR /seqr
 
 # install seqr dependencies
-RUN pip install wheel && pip install -r requirements.txt
+RUN pip install --no-cache-dir wheel && pip install --no-cache-dir -r requirements.txt
 
 ARG SEQR_SERVICE_PORT
 ENV SEQR_SERVICE_PORT=$SEQR_SERVICE_PORT
```

Comment:
@hanars @ShifaSZ Do either of you see any value in making the seqr service port a variable like this? I'm not sure I see a case where this would need to change, and propose getting rid of this and setting the expose port to 8000 statically.

If someone's running the container in an infrastructure where using port 8000 is problematic, the "right" way to deal with that is for them to change the port on the kubernetes Service, or publish a different port with the docker `-P` flag.

Reply:
In other words, the `EXPOSE` directive should always be 8000 as long as gunicorn is configured to listen on that port in https://github.com/broadinstitute/seqr/blob/master/deploy/docker/seqr/config/gunicorn_config.py#L2 -- setting it to anything else won't work.

Reply:
The only reason I can think of to make this configurable is if someone has a local installation where they need to change the port, but as long as thats something they can change via how they run docker compose, I'm fine to make this static